app/jobs/process_metrics_job.rb

```diff
@@ -0,0 +1,12 @@
+##
+# This job processes the Github events previously stored, generates the
+# metrics and leaves them in a ready to use state.
+class ProcessMetricsJob < ApplicationJob
+  queue_as :default
```

Comment:
ActiveJob retries jobs by default. Should this job be retried in case of failure? 🤔 I think maybe not.. I'd use `discard_on StandadError` and log the error (or notify it to any exception service we might have)
https://edgeapi.rubyonrails.org/classes/ActiveJob/Exceptions/ClassMethods.html#method-i-discard_on